### PR TITLE
chore: Add issue template

### PR DESCRIPTION
**Pull request by tido64**

### CODE REVIEW

**Review by kelset — COMMENTED** [review]
I think it's fine, but if it was me I'd probably move a couple things around - from:
* Description
* Env
* Steps
* Expec
* Actual

To

* Your Env
* Actual Behaviour + expected
* Steps to repro

And I'd consider creating a different one that serves as a "fallback" (I think you can create one that is named ISSUE_TEMPLATE.md it will be served as a "catch all" option? can't remember precisely).

**Comment by sweggersen** [comment]
> I think it's fine, but if it was me I'd probably move a couple things around - from:
> 
> * Description
> * Env
> * Steps
> * Expec
> * Actual
> 
> To
> 
> * Your Env
> * Actual Behaviour + expected
> * Steps to repro
> 
> And I'd consider creating a different one that serves as a "fallback" (I think you can create one that is named ISSUE_TEMPLATE.md it will be served as a "catch all" option? can't remember precisely).

It seems you remembered correctly: https://github.community/t/default-issue-template/1193/10

**Comment by tido64** [comment]
> It seems you remembered correctly: https://github.community/t/default-issue-template/1193/10

The wording in the documentation seems to suggest that `issue_template.md` is considered legacy: https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser.

Regardless, I've made sure that potential contributors can still use a blank issue if desired.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+name: "Bug report \U0001F41B"
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+---
+
```
Comment (kelset) [inline .github/ISSUE_TEMPLATE/bug_report.md:8]:
```suggestion
 ### Description
```

I'd still leave the title tbh. Aside from this, all looking good for me 👍

**Review by kelset — APPROVED** [review]
LG

**Pull request merged**